x-pack/plugins/index_management/public/application/components/mappings_editor/lib/utils.ts

```diff
@@ -782,6 +777,19 @@ export function getStateWithCopyToFields(state: State): State {
   return updatedState;
 }
 
+function addChildFieldsToState(field: NormalizedField, state: State): State {
```

Comment:
any chance we can add a small test for this?

Reply:
Added a unit test for the function that calls this to cover that :)